docs/docs/rest-api/public/api/v2/types/residency.raml

```diff
@@ -0,0 +1,22 @@
+#%RAML 1.0 Library
+uses:
+  strings: stringTypes.raml
+types:
+  Residency:
+    type: object
+    description: |
+      When using local persistent volumes that pin tasks onto agents,
+      these values define how Marathon handles terminal states of these tasks.
+    properties:
+      relaunchEscalationTimeoutSeconds:
```

Comment:
I'm about 95% confident this field currently doesn't do anything. (for apps). And, it seems like it directly overlaps in functionality with `UnreachableStrategy.inactiveAfterSeconds` (which does work, however, is currently forced to be disabled, much to the dismay of @unterstein ).

As far as I can tell, it didn't do anything in as far back as Marathon 1.1.

I'd be inclined to just remove the field since it's been broken that long, and overlaps in functionality with UnreachableStrategy.